Chargebee/Classes/Purchase/CBPurchaseManager.swift

```diff
@@ -17,9 +17,16 @@ public class CBPurchase: NSObject {
     private var authenticationManager = CBAuthenticationManager()
     var productRequest: SKProductsRequestFactory = SKProductsRequestFactory()
 
-    private var restoredPurchasesCount = 0
+     var restoredPurchasesCount = 0
     private var activeProduct: SKProduct?
     var customer: CBCustomer?
+    
+     var restoreResponseHandler: ((Result<[InAppSubscription], RestoreError>) -> Void)?
+     var result: ReceiptResult<String>?
+     var refreshHandler: RestoreResultCompletion<String>?
+
+   public var includeNonActiveProducts = false
+    var restoreHandlerisActive = false
```

Comment:
- Minor Linting/Formatting issues.
- Declare all variables as private unless otherwise needed.

Reply:
Minor Linting/Formatting issues.and restore feature added in new extensions of same file so could not able to declare them as private